walkthroughs/week-4-tdd/project/src/main/java/com/google/sps/MeetingRequest.java

```diff
@@ -24,6 +24,9 @@
   // All the people that should be attending this new meeting. Use a set to avoid duplicates.
   private final Collection<String> attendees = new HashSet<>();
 
+  // Some optional attendees for this new meeting. Use a set to avoid duplicates.
+  private final Collection<String> optional_attendees = new HashSet<>();
```

Comment:
Is it too fancy to de-duplicate between attendees and optional_attendees?

Reply:
Not too fancy. Do you suggest a check in addOptionalAttendee or something more?

Reply:
I was thinking a simple check in addAttendee and addOptionalAttendee. Alternatively, this could be a good TDD example where you tell the students there's a small bug in the provided code and lead them to write a test that catches the issue, then fix it.

Reply:
Added a check in addOptionalAttendee. There is no "addAttendee", they have to be specified in the constructor.

Reply:
Whoops, should have read more closely. Sounds good!